test/unit/betterRules/compileTests.ts

```diff
@@ -0,0 +1,76 @@
+import { assert } from 'assertthat';
+import { BetterRulesRecord } from '../../../lib/betterRules/BetterRulesRecord';
+import { compile } from '../../../lib/betterRules/compile';
+
+suite('rulesCompiler', (): void => {
+  test('compiles a rule with a  boolean to a deactivated rule.', async (): Promise<void> => {
```

Comment:
There a redundant space in here.

```suggestion
  test('compiles a rule with a boolean to a deactivated rule.', async (): Promise<void> => {
```